cmd/server/main.go

```diff
@@ -3,33 +3,22 @@ package main
 import (
 	"fmt"
 	"log"
-	"net"
 	"strings"
 
-	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/storage/pkg/dapr"
 
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
 
-	"github.com/infobloxopen/atlas-app-toolkit/server"
-
 	"github.com/infobloxopen/atlas-app-toolkit/gorm/resource"
 )
 
 func main() {
-	doneC := make(chan error)
+	done := make(chan struct{})
 	logger := NewLogger()
-	if viper.GetBool("internal.enable") {
-		go func() { doneC <- ServeInternal(logger) }()
-	}
-
-	go func() { doneC <- ServeExternal(logger) }()
-
-	if err := <-doneC; err != nil {
-		logger.Fatal(err)
-	}
+	dapr.InitPubsub("Storage", "messages", ":50001", "3501", logger, done)
```

Comment:
Please, don't use hard-coded values. Use vars/consts instead

Reply:
Sorry for that, it was just for test.

Reply:
Done